packages/graphback-core/tests/subscriptionFilterTest.ts

```diff
@@ -0,0 +1,736 @@
+/* eslint-disable max-lines */
+import { Scalars, Maybe, QueryFilter, StringInput, IdInput, BooleanInput, IntInput, FloatInput, DateTimeInput } from '../src/runtime/QueryFilter';
+import { createSubscriptionFilterPredicate } from '../src/runtime/subscriptionFilter';
+
+type User = {
+  __typename?: 'User';
+  id?: Scalars['ID'];
+  name?: Scalars['String'];
+  verified?: Maybe<Scalars['Boolean']>;
+  age?: Scalars['Int'];
+  score?: Scalars['Float'];
+  createdAt?: Scalars['DateTime'];
+};
```

Comment:
Are these enough to cover the `ObjectID` case?

Reply:
I used all the typed filters were available in `QueryFilter.ts`:

https://github.com/aerogear/graphback/blob/c2a718aa9da5d00ab74877996f2a72bcc65c414d/packages/graphback-core/src/runtime/QueryFilter.ts#L17-L83

I'm guessing a PR will need to be created to update QueryFilter with ObjectID.

Reply:
Sure, we'll quickly need it bu happy to having this done in followup PR.